biohansel/subtype/subtyper.py

```diff
@@ -1,21 +1,22 @@
 # -*- coding: utf-8 -*-
 """
-Functions for subtyping of reads (e.g. FASTQ) and contigs (e.g. FASTA) using bio_hansel-compatible subtyping schemes.
+Functions for subtyping of reads (e.g. FASTQ) and contigs (e.g. FASTA) using biohansel-compatible subtyping schemes.
 """
 import logging
 from typing import Optional, List, Dict, Union, Tuple
 
 import pandas as pd
 import re
 
-from .aho_corasick import init_automaton, find_in_fasta, find_in_fastqs
-from .const import COLUMNS_TO_REMOVE
-from .qc import perform_quality_check, QC
-from .subtype import Subtype
-from .subtype_stats import SubtypeCounts
-from .subtype_stats import subtype_counts
-from .subtyping_params import SubtypingParams
-from .utils import find_inconsistent_subtypes, get_scheme_fasta, get_scheme_version, init_subtyping_params
+from biohansel.subtype.aho_corasick import init_automaton, find_in_fasta, find_in_fastqs
+from biohansel.subtype.const import COLUMNS_TO_REMOVE
+from biohansel.subtype.qc import perform_quality_check, QC
+from biohansel.subtype.subtype import Subtype
+from biohansel.subtype.subtype_stats import SubtypeCounts
+from biohansel.subtype.subtype_stats import subtype_counts
+from biohansel.subtype.subtyping_params import SubtypingParams
+from biohansel.subtype.util import get_scheme_fasta, get_scheme_version, init_subtyping_params
+from biohansel.utils import find_inconsistent_subtypes
 
 
 def subtype_reads_samples(reads: List[Tuple[List[str], str]],
```

Comment:
It can be user-defined. The default value is 1. 

See https://github.com/phac-nml/biohansel/pull/52/files/9511966bb5c1825cc60001ba162d0f5045a8751a#diff-6bd3773d4492443bd26fad15adf979b0R184

Reply:
Ok yes my mistake